README.md

```diff
@@ -1,2 +1,29 @@
-# annual-radiation
+# Annual Radiation
+
 Annual radiation recipe for Pollination
+
+This recipe calculate the annual radiation for each timestep provided by the wea file.
+The outputs are stored under `results/direct` and `results/total`. Nigh-time hours are
+filtered before running the simulation. To match the results for eah hours see the list
+of hours in sun-up-hours.txt.
+
+# Process
+
+This recipe calculates the total amount of Radiation by calculating the direct sunlight
+radiation from sun disks and adding them to the contribution from indirect sky radiation.
+
+```
+Total radiation = direct sun radiation + indirect sky radiation
+```
+
+The number of bounces for direct radiation is set to 0 and cannot be modified by user.
+You can adjust the number of bounces for indirect sky radiation.
+
+# Limitations
+
+This recipe is limited to calculating up to `9999` number of suns at a time. This means
```

Comment:
Looking forward to removing this!!!

Reply:
I tested it with the latest version and there was a bug! Greg has already fixed it. I'll test it when the new version is built tonight or tomorrow.